### PR TITLE
Add "Getting Approval" section with license check.

### PR DESCRIPTION
**Pull request by ktlim**

### CODE REVIEW

**Comment by timj** [comment]
Looks okay to me. @jonathansick when do we get travis validation of PRs in the dev guide?

File: build-ci/third_party.rst [hunk 1]
```diff
@@ -5,6 +5,19 @@ Distributing Third-Party Packages with EUPS
 This page documents how to make a third-party software package install-able
 using the :command:`eups distrib install` command.
 
+Getting Approval
+================
+
+Creating a new third-party package that will be a dependency of the LSST code
+typically requires an :ref:`RFC <decision-making-rfc`.  If the code is to be
```
Comment (jonathansick) [inline build-ci/third_party.rst]: Need a closing bracket on `<decision-making-rfc>`

**Comment by jonathansick** [comment]
I can set up Travis (and I guess might as well move to LTD?) by first-thing tomorrow morning.

**Comment by timj** [comment]
Whenever you want. It's your timeline. I was just interested in knowing the timeline (partly because of the lack of feedback on RST linting in the PR).

**Comment by jonathansick** [comment]
Yeah, I agree that feedback on reST errors will be hugely beneficial; especially with Edit on GitHub-type workflows.

**Comment by ktlim** [comment]
Darn.  I had that closing angle bracket before, but it probably got lost in a reformat.  Fixed.

**Comment by timj** [comment]
I've rebased this and it passes the travis tests. @ktlim shall we merge it?

**Pull request merged**